src/software/gui/full_system/threaded_full_system_gui.cpp

```diff
@@ -67,7 +67,7 @@ void ThreadedFullSystemGUI::createAndRunFullSystemGUI()
         world_draw_functions_buffer, ai_draw_functions_buffer, play_info_buffer,
         sensor_msg_buffer, view_area_buffer, worlds_received_per_second_buffer,
         primitives_sent_per_second_buffer, mutable_thunderbots_config);
-    full_system_gui->show();
+    full_system_gui->showFullScreen();
```

Comment:
this should be behind a command line arg or removed from this PR

Reply:
removed